ChangeLog

```diff
@@ -1,3 +1,31 @@
+Tue, 12 Mar 2024 11:33:49 +0100 IvozProvider Team <vozip+ivozprovider@irontec.com>
+
+    * IvozProvider 4.0.2 released
+
+    * Portals
+        - Fixed multiple select options only displaying 30 entries
+        - Fixed a bug that removed Active Calls after two minutes
+        - Fixed a bug that prevented some Active Calls to be removed when finished
+        - Fixed display issues when browser text language was not English or Spanish
+        - Updated dashboard icons, colors and texts
+        - Fixed multiple translations issues
+
+    * Provisioning
+        - Added missing Terminal provisioning microservice package
+        - Removed not allowed PHP functions from default templates
+
+    * Proxies
+        - Fixed SUBSCRIBE messages socket selection and allowed Events
```

Comment:
- Handle SUBSCRIBE in custom route to skip INVITE-only logics.

Reply:
Added